en/docs/services/create-your-first-service.md

```diff
@@ -1,177 +1,250 @@
+
+2. In the **Connection Name** field, enter `covid19Client`, and save.
+
+3. Click the last **+** icon in the low-code diagram and click **API Calls**.
 
-3. Click **API Calls** and complete the following steps to create a new HTTP connection:
+4. Under **Choose existing connection**, click **covid19Client**.
 
-    1. Click **HTTP** and enter the details as follows:
+5. In the **Operation** field, select **Country Status**. Then enter the following information in the other fields that appear.
+
+    | **Field**                 | **Value**         |
+    |---------------------------|-------------------|
+    | **Country**               | `country`         |
+    | **Response VariableName** | `statusByCountry` |
     
-        | **Field**           | **Value**                      |
-        |---------------------|--------------------------------|
-        | **Connection Name** | `httpEndpoint`                 |
-        | **URL**             | `https://api.covid19api.com`   |
+    Save the information.
     
-        ![Select HTTP Connection](../assets/img/services/select-http-connection.png){.cInlineImage-half}
-        
-    2. Click **Save**.
-        
-    3. Click the **+** icon below the HTTP API call you added. 
+6. Now let’s extract the current total cases count from the response and store it in a variable.
+
+    To do this, click the last **+** icon in the low-code diagram and click **Variable**.
     
-        ![Update Existing Connection](../assets/img/services/update-existing-connection.png){.cInlineImage-half}
-        
-    4. Click **API Calls** and then click the existing connection.
+    Enter information as follows:
     
-        ![Select Existing connection](../assets/img/services/select-existing-connection.png){.cInlineImage-half}
-        
-    5. Enter the details as follows:
-        
-        | **Field**               | **Value**                       |
-        |-------------------------|---------------------------------|
-        | **OPERATION**           | **get**                         |
-        | **Resource Path**       | `"/total/country/united-states"`|
-        | **Select Payload Type** | **JSON**                        |
-        
-        !!! info
-            To select **JSON** as the payload type, you must enable **Do you want to extract a payload?**.
-            ![Do you want to extract a payload button](../assets/img/services/enable-payload.png){.cInlineImage-half}
+    | **Field**      | **Value**                      |
+    |----------------|--------------------------------|
+    | **Type**       | `var`                          |
+    | **Name**       | `totalCases`                   |
+    | **Expression** | `statusByCountry?.cases ?: 0d` |
+    
+    Save the information.
+    
+## Step 3: Get the population data
 
-    6. Click **Save & Done**.
-        
-   Now you have designed the service to fetch the active COVID-19 cases in the United States.
+To get the population data:
```

Comment:
Why do we need to say this again as topic clearly mention the same?

Reply:
Updated this sentence to elaborate on the purpose of the procedure.